ui/pages/settings/networks-tab/index.scss

```diff
@@ -7,31 +7,34 @@
   }
 
   &__content {
-    display: flex;
-    height: 100%;
-    max-width: 779px;
-    justify-content: space-between;
+    &--with-networks-list-popup-footer {
+      // padding-bottom required to offset content from fixed footer: &__networks-list-popup-footer
+      padding-bottom: 80px;
+      // isolation contains stacking context(z-index) of &__networks-list-popup-footer
+      isolation: isolate;
+    }
```

Comment:
This is a conditional class that adds styles necessary to keep the content visible when a fixed footer is added and isolated the stacking context concerns to this div so the fixed footer will never overlap with any elements outside of `__content`

<img width="376" alt="Screen Shot 2022-06-15 at 8 40 58 AM" src="https://user-images.githubusercontent.com/8112138/173870837-0ad4e49c-e665-4e5e-aa85-af29ad1a2360.png">
<img width="410" alt="Screen Shot 2022-06-15 at 8 41 58 AM" src="https://user-images.githubusercontent.com/8112138/173870846-0a2f9918-4daa-43dc-8040-0e4ba891cd32.png">